Types/ELF/include/elf.hpp

```diff
@@ -0,0 +1,419 @@
+#pragma once
+
+#include "utils.hpp"
+
+namespace GView::Type::ELF
+{
+namespace Go
```

Comment:
GView::Type::ELF::Go ?

Reply:
https://github.com/gdt050579/GView/issues/66
for now.. gonna move it